spec.bs

```diff
@@ -4859,6 +4859,92 @@ perform the following steps. They return a [=batching scope=].
 
 </div>
 
+<div algorithm>
+To <dfn>determine whether to perform default contributeToHistogramOnEvent() processing</dfn>
+given a {{PrivateAggregation}} |this|, a {{DOMString}} |event| and a [=map=] (with {{DOMString}}
+keys) |contribution|, perform the following steps. They return a [=boolean=] or an [=exception=].
+1. Let |global| be |this|'s [=relevant global object=].
+1. Let |function| be |global|'s [=InterestGroupScriptRunnerGlobalScope/worklet function=].
+1. Let |origin| be |global|'s [=InterestGroupScriptRunnerGlobalScope/origin=].
+1. If |this|'s <a spec="private-aggregation-api" for="PrivateAggregation">
+  allowed to use</a> is false, return a {{TypeError}}.
+1. Set |contribution| to the result of [=converted to an IDL value|converting=]
+    |contribution|'s [=converted to a JavaScript value|JavaScript value=] to the
+    IDL type {{PAExtendedHistogramContribution}}, catching any [=exception=]

```

Comment:
`rethrowing any [=exceptions=]` should be fine, as that's what the Web IDL standard does in many places.